pymatgen/transformations/advanced_transformations.py

```diff
@@ -1360,3 +1361,213 @@ def inverse(self):
     @property
     def is_one_to_many(self):
         return False
+
+class CubicSupercellTransformation:
```

Comment:
This class doesn't subclass `AbstractTransformation`.

E.g. see `class SupercellTransformation(AbstractTransformation):` Note the `(AbstractTransformation)`.

Furthermore, it doesn't implement the required functions that all abstract transformations should. See `SupercellTransformation` for an example. E.g., you should add and modify the following:

```python
    def apply_transformation(self, structure):
        return structure * self.scaling_matrix

    def __str__(self):
        return "Supercell Transformation with scaling matrix " + \
            "{}".format(self.scaling_matrix)

    def __repr__(self):
        return self.__str__()

    @property
    def inverse(self):
        raise NotImplementedError()

    @property
    def is_one_to_many(self):
        return False
```

Read the `AbstractTransformation` docs for more information.